packages/core/src/amazonq/lsp/lspController.ts

```diff
@@ -373,6 +373,14 @@ export class LspController {
                 // only download LSP for users who did not turn on this feature
                 // do not start LSP server
                 await LspController.instance.tryInstallLsp(context)
+                // enable local workspace index ONCE for Amzn users
+                if (
+                    AuthUtil.instance.isAmznUser() &&
+                    globals.globalState.tryGet('aws.amazonq.workspaceIndexToggleOn', Boolean) !== true
```

Comment:
why not use the existing `workspaceIndex` setting? if the `workspaceIndex` setting is undefined, then do this "one-time" step. Then set `workspaceIndex` to true. Then the user can set it to false if they want.

Reply:
that setting has default value false. Let me double check if it is undefined

Reply:
try `isSet` https://github.com/aws/aws-toolkit-vscode/blob/246548b94e9ea688263e92884dca54ee27daff9c/packages/core/src/shared/settings.ts#L160-L170

Reply:
`isSet` does not work. `isSet` always return false,